swagger/v3/component_schemas/ECFEnrolment.yml

```diff
@@ -0,0 +1,117 @@
+description: "The details of an ECF Participant enrolment"
+type: object
+required:
+  - training_record_id
+  - email
+  - school_urn
+  - participant_type
+  - cohort
+  - training_status
+  - participant_status
+  - teacher_reference_number_validated
+  - eligible_for_funding
+  - pupil_premium_uplift
+  - sparsity_uplift
+  - schedule_identifier
+  - validation_status
+  - delivery_partner_id
+  - created_at
+properties:
+  training_record_id:
+    description: "The unique identifier of the participant training record"
+    type: string
+    format: uuid
+    example: "000a97ff-d2a9-4779-a397-9bfd9063072e"
+  email:
+    description: "The email address registered for this ECF participant"
+    type: string
+    example: "jane.smith@some-school.example.com"
+  mentor_id:
+    description: "The unique identifier of this ECF participants mentor"
+    type: string
+    nullable: true
+    example: bb36d74a-68a7-47b6-86b6-1fd0d141c590
+    format: uuid
+  school_urn:
+    description: "The Unique Reference Number (URN) of the school that submitted this ECF participant"
+    type: string
+    example: "106286"
+  participant_type:
+    description: "The type of ECF participant this record refers to"
+    type: string
+    example: ect
+    enum:
+      - ect
+      - mentor
+  cohort:
+    description: "Which cohort this ECF participant is associated with"
+    type: string
+    example: "2021"
+  training_status:
+    description: "The training status of the ECF participant"
+    type: string
+    example: active
+    enum:
+      - active
+      - deferred
+      - withdrawn
+  participant_status:
+    description: Replaces the old status field. Indicates if a SIT has advised DfE of a transfer or a withdrawal of the participant from the school
+    type: string
+    enum:
```

Comment:
Do we want to add 'left' as a value here. It applies once a leaving_date is passed.